### PR TITLE
#5-schema-fahad: Database Schema

### PR DESCRIPTION
**Pull request by Fahi28**
Initial draft for the schema based upon our ERD.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+CREATE TABLE earthquakes (
+    earthquake_id BIGINT GENERATED ALWAYS AS IDENTITY,
+    time TIMESTAMPTZ NOT NULL,
+    tsunami BOOLEAN,
+    felt_report_count SMALLINT,
+    cdi FLOAT,
+    latitude FLOAT NOT NULL,
+    longitude FLOAT NOT NULL,
+    detail_url VARCHAR(255) UNIQUE NOT NULL
```
Comment (S1mpySloth) [inline schema.sql:43]: Missing comma at the end here?

**Review by qrafiq1 — COMMENTED** [review]
Looks good, have you tested it on a local psql database?

File: schema.sql [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+CREATE TABLE users (
+    user_id BIGINT GENERATED ALWAYS AS IDENTITY,
+    email VARCHAR(255) UNIQUE NOT NULL,
+    phone VARCHAR(15) UNIQUE NOT NULL,
+    PRIMARY KEY user_id
+);
+
+CREATE TABLE regions (
+    region_id BIGINT GENERATED ALWAYS AS IDENTITY,
+    min_latitude FLOAT NOT NULL,
```
Comment (S1mpySloth) [inline schema.sql:64]: Using FLOAT for latitude and longitude is fine, but consider DECIMAL for greater precision maybe?

File: schema.sql [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+CREATE TABLE magnitude (
+    magnitude_id SMALLINT GENERATED ALWAYS AS IDENTITY,
+    magnitude_type VARCHAR(3) UNIQUE NOT NULL,
+    PRIMARY KEY magnitude_id
+);
+
+CREATE TABLE networks (
+    network_id SMALLINT GENERATED ALWAYS AS IDENTITY,
+    networK_name VARCHAR(2) UNIQUE NOT NULL,
```
Comment (jiuliangut) [inline schema.sql:25]: tiny typo - think you've made the K here upper case!

**Review by jiuliangut — CHANGES_REQUESTED** [review]
looks good - just a tiny typo to fix otherwise no obvious errors :)

**Review by jiuliangut — APPROVED** [review]
Nice work fixing typos quickly - looks great!

**Review by qrafiq1 — APPROVED** [review]
Works fine, well done.

**Pull request merged**